packages/test/test-version-utils/mocharc-common.cjs

```diff
@@ -4,7 +4,7 @@
  */
 
 "use strict";
-const options = import("./dist/compatOptions.js");
```

Comment:
This is the actually problematic line: `import` returns a promise to the contents of the module. So the subsequent logic in `getFluidTestVariant` wouldn't work.